### ISSUE TITLE
Dev branch lacks clear notice of Genome2d Dependency

### ISSUE THREAD

**Issue by greysondn** [FlixelCommunity/flixel#226]
I found out that we now have a strong dependency on Genome2d by trying to compile the FPT test-suite without it in my classpaths. However, there's nothing in the readme or such that says that it's needed.

Could we ship the binary for Genome2d in the source tree, by chance? Otherwise, at least noting that it's a dependency could save headaches.

**Comment by Dovyski** [comment]
Yeah, that's a problem. This dependency is a result of the new GPU render. The way I see things, we have two options:
- Merge the Genome2D SWC into the release Flixel SWC (the one available for download at our website).
- Include the Genome2D SWC in the source tree, documenting it somewhere (like http://learn.flixelcommunity.org).

Honestly I think we should do both.
